solr/solr-ref-guide/modules/query-guide/pages/faceting.adoc

```diff
@@ -149,7 +147,7 @@ Note that the default logic is changed when <<Limiting Facet with Certain Terms>
 |Optional |Default: `100`
 |===
 +
-This parameter specifies the maximum number of constraint counts (essentially, the number of facets for a field that are returned) that should be returned for the facet fields.
+The number of facets for a field that are returned.
```

Comment:
Yeah this doc uses `facets`, `terms`, `constraints`, `counts`, `buckets` all in kind of a similar way. How's this?

```suggestion
The number of facet counts returned.
```